src/telemetry/Manager.cc

```diff
@@ -38,6 +43,232 @@ void Manager::InitPostBrokerSetup(broker::endpoint& ep)
 	pimpl.swap(ptr);
 	}
 
+// -- collect metric stuff -----------------------------------------------------
+
+const std::vector<std::string_view> Manager::Metric::label_names() const
```

Comment:
```suggestion
 std::vector<std::string_view> Manager::Metric::label_names() const
```

No need for `const` restriction here, it's copied/moved anyways.

Reply:
The whole method is gone after removing the accessors.